.travis.yml

```diff
@@ -221,4 +224,4 @@ deploy:
   on:
     repo: ${GITHUB_REPO_NAME}
     tags: true
-    condition: "$TRAVIS_TAG == *a*"
+    condition: "$TRAVIS_TAG != *[ab]*"
```

Comment:
@matt-chan Can you check this? There should be no condition at all, all releases (alpha, beta and main) can be uploaded to conda. They will all end up in different channels, such that there should be no accidental installs of alpha or beta versions by users that follow the main channel. It also seemzs that there is a mistake in the script line above. It misses the `-l` before the label, which is probably causing related problems.